packages/forms/config/forms.php

```diff
@@ -26,6 +26,14 @@
 
         ],
 
+        'repeater' => [
```

Comment:
Please remove these config options, instead add `reorder ableWithButtons()` and `isReorderableWithButtons()` methods to the Repeater and Builder classes

Reply:
done